subcommittees/mentoring/checklists-discussion-sessions.md

```diff
@@ -1,63 +1,31 @@
+> on [this page](procedure-discussion-session.md).  
 
 Discussion sessions are organized and conducted through this etherpad: 
 
 [http://pad.software-carpentry.org/instructor-discussion](http://pad.software-carpentry.org/instructor-discussion)
 
-### In advance
-- [ ] Sign up for a slot on the etherpad. 
-
-### Before the discussion (day of)
-- [ ] If necessary, split the participants into two groups.  
-	* Rule of thumb: split when the number of workshops represented is greater than 5.
-	* There is a second BlueJeans room on the etherpad if you need to split the group. 
-	* Try to balance the number of people de-briefing, getting help, and checking out between the two groups.  
-- [ ] Coordinate with your host/co-host about who will lead and who will take notes
-
-### During the discussion
-- [ ] Sign on and lead the meeting
-	- You can use the agenda at the bottom of the etherpad as a starting point/outline for discussion, as you choose
-	- Don't feel like you have to answer all the questions - draw on the discussion 
-	participants to answer each other's questions.  
-	- If someone is attending as part of their training checkout, prompt them to 
-ask questions.  
-- [ ] Take notes, if possible, about meeting discussion
-
-### After the discussion
+## Leading the Discussion
+- [ ] Sign on as close to the start time as you can.  
+- [ ] Confirm that your fellow host is present (if there is one) and who will be leading/taking notes
+- [ ] (Optional) If a group is very large and you have a co-host, you can split the group in two, using the extra BlueJeans link.  
+- [ ] Copy the agenda from the bottom of the etherpad to the space under your time slot
+- [ ] Lead the discussion
+	- We recommend following the main points of the agenda, but feel free to let discussion go where people want.  Not all the questions need to be answered by everyone.  
+	- Don't feel like *you* have to answer all the questions as the host - draw on the 
+	other discussion participants to answer each other's questions.  
+	- If someone is attending as part of their training checkout, prompt them to ask a question. 
+	- Try to leave enough time at the end for the final agenda item.  
+- [ ] Take notes
+
+## After the discussion
 - [ ] Archive the etherpad by clicking on the star in the top right corner.  
 - [ ] Copy your notes to [this google doc](https://docs.google.com/document/d/1zQTkNCev9AzAlA464jUa8ifFaTx_8mb1NX6xvUbaGvM/edit) and label with the date and which session you led.  
-- [ ] Update AMY to reflect which trainees participated in the disucussion.
-	1. Log in to AMY.
-	2. More... Trainees.
-	3. Put the person's name in the "Filter" box in the upper left.
-	4. Click the white-and-gray [+] under "Progress" beside their name.
-	5. Make sure "Evaluated by" is your name.
-	6. Change "Type" to "Discussion".
-	7. Click "passed" or "failed" to reflect their participation.
-	8. Submit.
+- [ ] Email checkout@carpentries.org with the names and affiliations of 
```

Comment:
That would be a question for @ErinBecker and @maneesha, but I think it's good to have the same process that we currently have for the trainers to sign off on demos.